ui/src/organizations/components/AddMembersForm.tsx

```diff
@@ -5,11 +5,13 @@ import React, {PureComponent} from 'react'
 import {Button, ButtonType} from '@influxdata/clockface'
 import {Form, Grid} from 'src/clockface'
 import SelectUsers from 'src/organizations/components/SelectUsers'
-import {User} from '@influxdata/influx'
+
+// Types
+import {UserObject} from 'src/organizations/components/Members'
```

Comment:
We might be able to use a `Map` here? It could be typed as `Map<User['id'], User>`

Reply:
I prefer the use of object with key and value here for the users